test/test_utils/__init__.py

```diff
@@ -108,6 +110,10 @@ def is_dlc_cicd_context():
     return os.getenv("BUILD_CONTEXT") in ["PR", "CANARY", "NIGHTLY", "MAINLINE"]
 
 
+def is_benchmark_dev_context():
+    return is_pr_context() and ENABLE_BENCHMARK_DEV_MODE
```

Comment:
Does PR context and dev mode need to be enabled for benchmark dev mode to work? What about running locally?

Reply:
Good point. I fail to consider the local test scenario. Will remove pr_context restriction here.